solutions/cloud/hybrid/helpers/calm-dr-vm-tracking-scripts/pre-migration-script.py

```diff
@@ -172,4 +175,4 @@ def main():
         log.info("Exception: %s" % e)
         raise
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
ditto

Reply:
The commits did not address this one and the previous main() comment. Trivial, but reducing churn is nice when we can get it

Reply:
Done.